.github/workflows/ci.yml

```
@@ -363,19 +565,180 @@ jobs:
+        test-focused,
+        test-full,
+        security,
+        commitlint,
+        bundle,
+      ]
     if: always()
     steps:
-      - name: Check Status
+      - name: Checkout
+        uses: actions/checkout@v4
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)